src/main/java/it/pagopa/interop/probing/eservice/registry/updater/service/EserviceService.java

```diff
@@ -29,33 +11,17 @@
 import it.pagopa.interop.probing.eservice.registry.updater.util.EserviceState;
 import it.pagopa.interop.probing.eservice.registry.updater.util.EserviceTechnology;
 
-/**
- * The Class EserviceService.
- */
 public class EserviceService {
```

Comment:
Use interface

Reply:
this topic was already discussed in the previous pr